src/components/external-events/Properties.svelte

```diff
@@ -0,0 +1,29 @@
+<svelte:options immutable={true} />
```

Comment:
I tried to, but doing so required a _lot_ of additional overhead with type definitions that did not make much sense in translating from activity conventions to external event conventions. As such, a new class had to be created. It is possible in the future when a more comprehensive schema for external events is developed that we can maybe merge these two classes, but even so that may require a decent refactor. As we just don't have enough at this stage to enact that, a new, lighter class was created for External Events.

Reply:
Based on another conversation, we can commit to retiring this component in favor of `Parameters` once we swap to using `FormParamters` instead of `ExternalEventProperty`!

Reply:
Yep! Issue created: https://github.com/NASA-AMMOS/aerie-ui/issues/1463